confidence_checks/synthetic_model.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+
+from models import model_interface
+from data import recourse_adapter
+
+
+class SyntheticModel(model_interface.Model):
```

Comment:
Consider renaming file and class `ToyModel` as per previous comment on synthetic_data.py.